analysis/tc_apache_wicket.go

```diff
@@ -22,16 +22,46 @@ var ApacheWicket = TC{
 		"cloud-readiness",
```

Comment:
Lines 16-22 can be removed. If I am not mistaken those are leftovers from windup analysis

Reply:
@mguetta1 , done "(y)"